seismostats/analysis/bvalue/classic.py

```diff
@@ -4,6 +4,23 @@
 from seismostats.analysis.bvalue.utils import beta_to_b_value
 
 
+def _mle_estimator(magnitudes: np.ndarray,
+                   mc: float,
+                   delta_m: float,
+                   weights: np.ndarray | None = None) -> float:
+    '''
+    Internal function for the classic b-value estimator. For b-value
+    estimation use `ClassicBValueEstimator` instead.
```

Comment:
Should we include Args and Return here?

Reply:
I thought about it, but it is an internal function which should only be used if you know what they are... they are documented in the ClassicBValueEstimator